.github/CONTRIBUTING.md

```diff
@@ -1,338 +1,22 @@
-
-Currently, a presence goes through 3 separate stages of checks. All of these checks help the reviewers determine whether your presence is suitable for deployment.
-
-- `DeepScan` is a bot that checks for code quality. If you ever receive errors for new issues, you are **required** to fix them. *Warning: DeepScan doesn't always give you errors. Please look at CodeFactor warnings instead.*
-- `CodeFactor` is a bot that checks for code quality. If you ever receive errors for new issues, you are **required** to fix them.
-- `Schema Validation` will scan your `metadata.json` file for any errors (for e.g., missing fields, invalid value types, etc.). If you ever see any new issues, you are also **required** to fix those. Adding a schema field to your `metadata.json` file will allow your text editor (if supported) to show you these errors during development.
-
-## `Additional Rules`
-
-- **Always** make sure to start your presence in the most appropriate folder, if its name starts with _any_ Latin letter then it must be under its alphabetical match (for e.g., `D/dアニメストア` or `G/Google`). Any other Unicode/non-Latin characters **must** be under the `#` folder (for e.g., `#/巴哈姆特`) and numbers under the `0-9` folder (for e.g., `0-9/4anime`).
-
-After meeting all of the guidelines with the proper reviews and checks, your presence will be merged with the store.
-
-# Suggestions
-
-If you have some suggestions about our guidelines, you should contact us @ [PreMiD's Discord server](https://discord.premid.app) and we will check them!
-
-# Contributions
-
-`Revision 3` of the guidelines was written and was contributed to by the following individuals:
-
-<div>
-<a href="https://github.com/PreMiD"><img src="https://github.com/PreMiD.png?size=2048" width="48px" style="max-width:100%; border-radius: 50%;"/></a>
-</div>
-
-`Revision 2` of the guidelines was written and was contributed to by the following individuals:
-
-<div>
-<a href="https://github.com/CobyPowers"><img src="https://github.com/CobyPowers.png?size=2048" width="48px" style="max-width:100%; border-radius: 50%;"/></a>
-</div>
-
-`Revision 1` was maintained by the following individuals:
-
-<div>
-<a href="https://github.com/CobyPowers"><img src="https://github.com/CobyPowers.png?size=2048" width="48px" style="max-width:100%; border-radius: 50%;"/></a>
-<a href="https://github.com/Bas950"><img src="https://github.com/Bas950.png?size=2048" width="48px" style="max-width:100%; border-radius: 50%;"/></a>
-<a href="https://github.com/i1u5"><img src="https://github.com/i1u5.png?size=2048" width="48px" style="max-width:100%; border-radius: 50%;"/></a>
-</div>
+ - **Use commitlint**. For more information regarding this, please see [COMMIT_CONVENTION.md](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
+ - **Use the imperative tone** - "chore: update assets" > "chore: updates"
+ - **Use the present tense** - "feat: add feature" > "feat: added feature"
```

Comment:
this commit pains me
```suggestion
 - **Use the present tense** - "feat: add support for X" > "feat: added support for X"
```

Reply:
I don't care, it is just a really basic template

Reply:
having feat and feature is not good commit practice. Just apply it